ingest.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 # this script is for a CRON job ingest
-# no input folder is given as it assumes oracc-rest/neo
+# no input folder is given to the python call as it assumes oracc-rest/neo
 # if this is not correct just add a path after ingest.bulk_upload
 # eg. ingest.bulk_upload ingest/assets/dev/sample-glossaries/*
+
 cd /home/rits/oracc-rest
 source venv/bin/activate
-python3 -m ingest.bulk_upload
-echo "ingest complete"
+
+# Run the ingest script and capture the outcome in journal
+systemd-cat -t cron-ingest python3 -m ingest.bulk_upload
```

Comment:
Ah, this isn't quite what I was meaning. If bulk_upload produces no output then there will be nothing in the log, so no evidence that this script ran at all, since the "Ingest complete" line doesn't go through systemd-cat. If bulk_upload must always produce output then perhaps this isn't a problem.
I was thinking that whoever wanted to run this could prefix by systemd-cat -t cron-ingest and then we wouldn't have to do anything; this is the normal way.

Reply:
ah
The bulk-upload does provide some output: it writes the name of the file it's about to index and then when it's finished doing so, does that improve its use?